salt/buildbot/config/nginx.conf.jinja

```diff
@@ -21,18 +21,13 @@ server {
   error_log /var/log/nginx/buildbot-master.error.log;
   access_log /var/log/nginx/buildbot-master.access.log main;
 
-  rewrite ^/$ https://www.python.org/dev/buildbot/ redirect;
-  rewrite ^/3.7(/?)$ /all/#/grid?branch=3.7 redirect;
-  rewrite ^/3.8(/?)$ /all/#/grid?branch=3.8 redirect;
-  rewrite ^/3.9(/?)$ /all/#/grid?branch=3.9 redirect;
-  rewrite ^/3.10(/?)$ /all/#/grid?branch=3.10 redirect;
-  rewrite ^/3.x(/?)$ /all/#/grid?branch=main redirect;
-  rewrite ^/3.7.stable(/?)$ /all/#/grid?branch=3.7&tag=stable redirect;
-  rewrite ^/3.8.stable(/?)$ /all/#/grid?branch=3.8&tag=stable redirect;
-  rewrite ^/3.9.stable(/?)$ /all/#/grid?branch=3.9&tag=stable redirect;
-  rewrite ^/3.10.stable(/?)$ /all/#/grid?branch=3.10&tag=stable redirect;
-  rewrite ^/3.x.stable(/?)$ /all/#/grid?branch=main&tag=stable redirect;
-  rewrite ^/stable(/?)$ /all/#/grid?tag=stable redirect;
+  rewrite ^/3.(\d+)(/?)$ /#/grid?branch=3.$1 redirect;
+  rewrite ^/3.x(/?)$ /#/grid?branch=main redirect;
+  rewrite ^/3.(\d+).stable(/?)$ /#/grid?branch=3.$1&tag=stable redirect;
+  rewrite ^/3.x.stable(/?)$ /#/grid?branch=main&tag=stable redirect;
+  rewrite ^/stable(/?)$ /#/grid?tag=stable redirect;
+
+  rewrite ^/all/(.*)$ /$1 redirect;
 
   location @backend {
```

Comment:
Actually, I think this might be better now:

```suggestion
  location / {
```

(along with removing the other `location /` section below)

Reply:
that works i think!

Reply:
Done.